### PR TITLE
Feature/ivoice1/rohit

### LINKED ISSUE #7: Create an Invoice with no items
Acceptance Criteria : validate if the Fee is Zero

### LINKED ISSUE #8: Create an Invoice with single Flat Fee Item
Acceptance Criteria: Validate Total Fee is equal to Flat fee item.[]

### LINKED ISSUE #10: Create an Invoice with 1 Rate based item with Quantity and 1 Flat Fee Item
Acceptance Criteria: Validate Total fee of invoice is Quantity * Rate Fee + Flat Fee Item

### LINKED ISSUE #11: Create an Invoice with multiple items of same description
Acceptance Criteria: Validate number of line items on Invoice.

### PR DESCRIPTION
**Pull request by ranjanr2**
Pull Request for Issue Number 7,8,10

### CODE REVIEW

File: hs_err_pid7604.log [hunk 1]
```diff
@@ -0,0 +1,833 @@
+#
```
Comment (zxander-D) [inline hs_err_pid7604.log]: Please remove the log file from commit.

**Review by zxander-D — CHANGES_REQUESTED** [review]
please remove the log files on these commits

File: src/main/java/com/hundred/percent/capstone/Invoicify/invoice/controller/InvoiceController.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    @Autowired
+    InvoiceService invoiceService;
+
+    @GetMapping
+    public List<InvoiceDTO> getAllInvoices(){
+        return invoiceService.getAllInvoices();
+    }
+
+    @PostMapping
```
Comment (zxander-D) [inline src/main/java/com/hundred/percent/capstone/Invoicify/invoice/controller/InvoiceController.java:24]: consider adding an endpoint that explicitly say that you are adding an invoice 
ex: /addInvoice

**Review by zxander-D — DISMISSED** [review]

**Review by 404-bug-not-found — CHANGES_REQUESTED** [review]
Build is failing for code coverage. Can you please run the JaCoCo code coverage report and you will see, which class is having less code coverage and accordingly you can add additional tests to cover it.

**Review by zxander-D — APPROVED** [review]
LGTM

**Review by 404-bug-not-found — APPROVED** [review]

**Pull request merged**